ui/components/AutomationsTable.tsx

```diff
@@ -22,13 +23,20 @@ type Props = {
 };
 
 function AutomationsTable({ className, automations, hideSource }: Props) {
+  const { data } = useFeatureFlags();
+  const flags = data?.flags || {};
+
   automations = automations.map((a) => {
     return { ...a, type: removeKind(a.kind) };
   });
+
   const initialFilterState = {
     ...filterConfig(automations, "type"),
     ...filterConfig(automations, "namespace"),
     ...filterConfig(automations, "clusterName"),
+    ...(flags.WEAVE_GITOPS_FEATURE_TENANCY === "true"
```

Comment:
This can be cleaned up by removing the ternary from this block:

```suggestion
if (flags.WEAVE_GITOPS_FEATURE_TENANCY) {
  initialFilterState = {...initialFilterState, ...filterConfig(automations, "tenant") }
}
```
`inifialFilterState` needs to be a `let` instead for this to work.